Firebase/DynamicLinks/FIRDynamicLinks.m

```diff
@@ -117,8 +115,7 @@ @implementation FIRDynamicLinks {
 #ifdef FIRDynamicLinks3P
 
 + (void)load {
-  [FIRApp registerAsConfigurable:self];
-  [FIRComponentContainer registerAsComponentRegistrant:self];
+  [FIRApp registerLibrary:(id<FIRLibrary>)self withName:@"dynlinks" withVersion:kFIRDLVersion];
```

Comment:
The cast shouldn't be required since it should be `Class<FIRLibrary>` not `id<FIRLibrary>`.

Reply:
Fixed.